ci-operator/config/ovirt/csi-driver/ovirt-csi-driver-master.yaml

```diff
@@ -0,0 +1,21 @@
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+  - dockerfile_path: Dockerfile
+    from: base
```

Comment:
You refer a `base` image but you do not tag it from anywhere, this will fail to build.